configuration/cardano/shelley-dummy-genesis.json

```diff
@@ -0,0 +1,41 @@
+{
+    "activeSlotsCoeff": 0.99,
+    "protocolMagicId": 838299499,
```

Comment:
Redundant, we only use `networkMagic` since yesterday, as they always had to be equal (so this was wrong here).